### DISCUSSION TITLE
Why does pytest needs to connect to a redis ?

### DISCUSSION THREAD

**Discussion by Andromelus** [pytest-dev/pytest#8692] [Q&A]
- pytest 6.1.0
- python 3.5.2

(Stuck with these, so upgrade is not a solution)

When running a very simple test (code bellow), pytest actually does two tests (as I understand the logs). The actual test and some connexion to Redis test.

## The test

```
import pytest


class TestSample:
    def test_assert_toto(self):
        """[summary]
        """
        assert type("") == str
```

When running the following command:

` pytest -v hadoop/test_sample.py`

The output is the following (complete logs/stacktrace in attached file):
```

============================= test session starts ==============================
platform linux -- Python 3.5.2, pytest-6.1.0, py-1.10.0, pluggy-0.13.1 -- /usr/local/bin/python3
cachedir: .pytest_cache
rootdir: /usr/share/project/projecttools, configfile: pytest.ini
plugins: addok-1.0.2
collecting ... collected 1 item

hadoop/test_sample.py::TestSample::test_assert_toto PASSED               [100%]
hadoop/test_sample.py::TestSample::test_assert_toto ERROR                [100%]

==================================== ERRORS ====================================
_______________ ERROR at teardown of TestSample.test_assert_toto _______________
```

[tests_results.txt](https://github.com/pytest-dev/pytest/files/6522525/tests_results.txt)

-----

- Why is there something doing another thing with redis at teardown ?
- How can I avoid it, or at least suppress its logs ?

Thanks ! 😃

**Comment by RonnyPfannschmidt** [comment]
at first  glance this is based on the addok plugin, i wasn't even aware it exists, please check with the maintainers of that plugin

**Reply by Andromelus** [reply]
Do you mean this https://github.com/addok/addok ?

Edit: seems like so. Will do.
```
sh-4.2# pip show addok
Name: addok
Version: 1.0.2
Summary: Search engine for address. Only address.
Home-page: https://github.com/addok/addok
Author: Yohan Boniface
Author-email: yohan.boniface@data.gouv.fr
License: WTFPL
Location: /usr/local/lib/python3.5/site-packages
Requires: ngram, hashids, Unidecode, progressist, redis, hiredis, python-geohash, falcon
```

**Reply by Andromelus** [reply]
As a "fun" fact, when uninstalling addok and re running the tests, everything works fine... 🤷🏻
